PIconnect/time.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+from typing import Union
```

Comment:
Could you remove this import for now to keep Python 2.7 support?

Reply:
sure